consensus/polybft/extra.go

```diff
@@ -88,46 +95,46 @@ func (i *Extra) UnmarshalRLPWith(v *fastrlp.Value) error {
 		return err
 	}
 
-	if num := len(elems); num != 4 {
-		return fmt.Errorf("not enough elements to decode extra, expected 4 but found %d", num)
+	if num := len(elems); num != 5 {
+		return fmt.Errorf("incorrect elements count to decode Extra, expected 5 but found %d", num)
 	}
```

Comment:
Just curious question. Checkpoint in Extra is required always? Please note it won't be able to encode the extra generated before.

Reply:
> Checkpoint in Extra is required always?

Yes, so that we have all the checkpoint data on disposal for sending to the rootchain/Ethereum on arbitrary block.

> Please note it won't be able to encode the extra generated before.

:ok_hand: aware of it, but it shouldn't be the problem, since if I'm correct, PoS v3 chain is going to be run from the genesis block.